cmake/dependencies/ABLASTR.cmake

```diff
@@ -161,15 +161,15 @@ set(ImpactX_openpmd_src ""
 set(ImpactX_ablastr_repo "https://github.com/ECP-WarpX/WarpX.git"
     CACHE STRING
     "Repository URI to pull and build ABLASTR from if(ImpactX_ablastr_internal)")
-set(ImpactX_ablastr_branch "24.10"
+set(ImpactX_ablastr_branch "25.01"
     CACHE STRING
     "Repository branch for ImpactX_ablastr_repo if(ImpactX_ablastr_internal)")
 
 # AMReX is transitively pulled through ABLASTR
 set(ImpactX_amrex_repo "https://github.com/AMReX-Codes/amrex.git"
     CACHE STRING
     "Repository URI to pull and build AMReX from if(ImpactX_amrex_internal)")
-set(ImpactX_amrex_branch "e64ffef57a7608d1d60f9abe738cc634e9c1272e"
+set(ImpactX_amrex_branch ""
```

Comment:
To clarify for my understanding:  without a specification here, what version of AMReX will be used?

Reply:
Without specifying it here, we pick the version set in ABLASTR (WarpX):
https://github.com/ECP-WarpX/WarpX/blob/development/cmake/dependencies/AMReX.cmake#L293-L301

I usually only overwrite it here if we temporarily need a newer version than what is set there.